lib/cmd/fh3/app/stage.js

```diff
@@ -19,12 +19,23 @@ base.describe['gitRef.value'] = i18n._("The name of the branch you'd like to dep
 base.alias.runtime = "r";
 base.alias.clean = "c";
 base.alias.autodeploy = "d";
+
+//Add custom example
+base.examples[0].cmd = 'fhc app stage --app=<appGuid> --env=<environmentName> (Defaults to branch:master)';
```

Comment:
@wtrocki I see the problem now. `desc` is part of the examples array.
Something like this fixes it - `base.examples[0].desc = i18n._("Stages a cloud application. Defaults to master branch");` <- Still ugly :)